deps/rabbitmq_shovel/src/rabbit_local_shovel.erl

```diff
@@ -437,6 +438,11 @@ add_routing(Msg0, Dest) ->
 status(_) ->
     running.
 
+pending_count(#{source := #{current := #{unacked_message_q := UAMQ}}}) ->
```

Comment:
Pending messages in AMQP0.9.1 are counted in the destination side, thus in local shovels they should be dest -> unconfirmed